src/app/user/user.service.ts

```diff
@@ -185,6 +185,17 @@ export class UserService {
       })
   }
 
+  /**
+   * Send email verification link to user.
+   */
+  sendEmailVerificationLink(): Observable<any> {
+    return this.http
+      .post(this.usersUrl + '/verificationcode', null, { headers: this.headers })
+      .map(response => {
```

Comment:
Please annotate response with it's correct type. E.g. `(response: Response)` if it's a Response.